### DISCUSSION TITLE
Does 3.18 not working on WIN7 anymore?

### DISCUSSION THREAD

**Discussion by acknowledge14** [acemod/ACE3#10381] [Q&A] [answered]
test with newest workshop ace 3.18 and github 3.18rc1 in win7, all report errors in rpts when play a mission:

`16:15:04 [CBA] (versioning) INFO: [2106,125.705,0] VERSIONING:cba=3.18.0.241008, ace=3.18.0.90`
`16:15:04 [ACE] (common) INFO: ACE is version 3.18.0.90 - CBA is version 3.18.0.241008 (min required 3.16.0)`
`16:15:04 Call extension 'ace' could not be loaded: Ւ²»µ½ָ¶¨µĳ̐򡣍`
`16:15:04 [ACE] (common) ERROR: Extension not found. [Return Code: -1]`

downloaded github ace 3.17.1, all work fine.

If so, I may have to update to win10 on my potato PC now.

**Comment by J-Kiel** [comment]
Also reporting the same error on my client, running Windows 7.  Haven't tried it on the server running Windows 7 yet.

18:13:21 [CBA] (versioning) INFO: [917,47.754,0] VERSIONING:cba=3.18.0.241008, zen=1.15.1.36, ace=3.18.0.90, acre=2.12.0.1056
18:13:22 [ACE] (common) INFO: ACE is version 3.18.0.90 - CBA is version 3.18.0.241008 (min required 3.16.0)
18:13:22 Call extension 'ace' could not be loaded: The specified procedure could not be found.

18:13:22 [ACE] (common) ERROR: Extension not found. [Return Code: -1]
18:13:22 [ACE] (nametags) INFO: ACRE Detected.
18:13:22 [CBA] (xeh) INFO: [917,48.153,0] PostInit finished.
18:13:24 [ACRE] (sys_core) INFO: AI Detection Activated.
18:13:24 [ACRE] (sys_signal) INFO: Using radio propagation model: LOS Multipath
18:13:24 [ACE] (common) INFO: Settings initialized.
18:13:24 [ACE] (common) INFO: 3 delayed functions running.
18:13:24 Call extension 'ace' could not be loaded: The specified procedure could not be found.

18:13:24 [ACE] (advanced_ballistics) ERROR: Error initializing map

**Comment by PabstMirror** [comment, marked as answer]
Looks like the new rust extension does not support windows 7
It might be possible to build if we set a custom target?
https://doc.rust-lang.org/rustc/platform-support/win7-windows-msvc.html

**Comment by jokoho48** [comment]
Windows 7 support stopped in 2020, and Steam has not supported Windows 7 since January 1, 2024. I don't think ACE needs to support Windows 7.

**Comment by always-up-app** [comment]
OK......
